code/game/objects/items/devices/lighting/flare.dm

```diff
@@ -13,33 +14,33 @@
 	var/on_damage = 7
 	var/produce_heat = 1500
 	light_wedge = LIGHT_OMNI
+	power_use = FALSE
 	activation_sound = 'sound/items/flare.ogg'
+	toggle_sound = null
 	drop_sound = 'sound/items/drop/gloves.ogg'
 	pickup_sound = 'sound/items/pickup/gloves.ogg'
 
-	var/overrides_activation_message = FALSE
-
 /obj/item/device/flashlight/flare/Initialize()
 	. = ..()
-	fuel = rand(400, 600)
+	fuel = rand(4 MINUTES, 6 MINUTES)
+
+/obj/item/device/flashlight/flare/Destroy()
+	. = ..()
+	STOP_PROCESSING(SSprocessing, src)
```

Comment:
This looks runtime-proned?

If the parent chain of destroys sleeps and the processing kicks in, it could try to run on an half-destroyed object, potentially runtiming it at worst and wasting tick time at best, my suggestion is:

```suggestion
/obj/item/device/flashlight/flare/Destroy()
	STOP_PROCESSING(SSprocessing, src)
	..()
```

Reply:
We probably do not need this at all since it's on the base class. I'll just remove this entirely.